### PR TITLE
webdriverio: add source-map-support package to deps

### LINKED ISSUE #3182: Not Getting Started [OS/X]
**Environment (please complete the following information):**
 - **WebdriverIO version:** 5.0.3
 - **Mode:** Standalone
 - **Node.js version:** 11.4.0
 - **NPM version:** 6.4.1
 - **Browser name and version:** FireFox 64.0 (64-bit)
 - **Platform name and version:** macOS 10.13.6
 - **Additional wdio packages used (if applicable):** none

**Config of WebdriverIO**
Followed the Getting started guide:
1. npm install webdriverio
2. Creat the specified test file

**Describe the bug**
running `node test` results in the following:

```
internal/modules/cjs/loader.js:605
    throw err;
    ^

Error: Cannot find module 'source-map-support/register'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:603:15)
    at Function.Module._load (internal/modules/cjs/loader.js:529:25)
    at Module.require (internal/modules/cjs/loader.js:659:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/andrewgibson/src/webdriverio-test/node_modules/webdriverio/build/index.js:8:1)
    at Module._compile (internal/modules/cjs/loader.js:723:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:734:10)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
```

**To Reproduce**
Follow the getting started guide: https://webdriver.io/docs/gettingstarted.html

**Expected behavior**
this should output the following:

`Title was: WebdriverIO · Next-gen WebDriver test framework for Node.js`

**Additional context**
I tried to resolve this issue by manually installing source-map-support. This allowed the test to execute for the first time. However, I then receive a different problem. Happy to break this out into another bug report if necessary.

**Logs after manually installing source-map-support**

***webdriver***
```
node test
2018-12-26T10:50:35.157Z DEBUG wdio-config: wdio-sync not found, running tests asynchronous
Title was: WebdriverIO · Next-gen WebDriver test framework for Node.js
2018-12-26T10:50:49.773Z ERROR webdriver: Request failed due to Error: {}DELETE /session/6f533b1f-2320-eb4b-b1a9-79ddf52406e6 did not match a known command
    at Request._callback (/Users/andrewgibson/src/webdriverio-test/node_modules/webdriver/src/request.js:86:27)
    at Request.self.callback (/Users/andrewgibson/src/webdriverio-test/node_modules/request/request.js:185:22)
    at Request.emit (events.js:189:13)
    at Request.EventEmitter.emit (domain.js:441:20)
    at Request.<anonymous> (/Users/andrewgibson/src/webdriverio-test/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:189:13)
    at Request.EventEmitter.emit (domain.js:441:20)
    at IncomingMessage.<anonymous> (/Users/andrewgibson/src/webdriverio-test/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:277:13)
    at IncomingMessage.emit (events.js:194:15)
Error: {}DELETE /session/6f533b1f-2320-eb4b-b1a9-79ddf52406e6 did not match a known command
    at Request._callback (/Users/andrewgibson/src/webdriverio-test/node_modules/webdriver/src/request.js:86:27)
    at Request.self.callback (/Users/andrewgibson/src/webdriverio-test/node_modules/request/request.js:185:22)
    at Request.emit (events.js:189:13)
    at Request.EventEmitter.emit (domain.js:441:20)
    at Request.<anonymous> (/Users/andrewgibson/src/webdriverio-test/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:189:13)
    at Request.EventEmitter.emit (domain.js:441:20)
    at IncomingMessage.<anonymous> (/Users/andrewgibson/src/webdriverio-test/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:277:13)
    at IncomingMessage.emit (events.js:194:15)
```

***selenium log***

```
./geckodriver --port 4444
1545821406466	geckodriver	INFO	geckodriver 0.21.0
1545821406473	geckodriver	INFO	Listening on 127.0.0.1:4444
1545821435477	mozrunner::runner	INFO	Running command: "/Applications/Firefox.app/Contents/MacOS/firefox-bin" "-marionette" "-foreground" "-no-remote" "-profile" "/var/folders/gv/r39k1_4s3r5gyrmbcmrrnp8r0000gn/T/rust_mozprofile.j1Ka9sN7y6p8"
1545821436308	addons.webextension.screenshots@mozilla.org	WARN	Loading extension 'screenshots@mozilla.org': Reading manifest: Invalid host permission: resource://pdf.js/
1545821436308	addons.webextension.screenshots@mozilla.org	WARN	Loading extension 'screenshots@mozilla.org': Reading manifest: Invalid host permission: about:reader*
1545821436454	Marionette	DEBUG	Received observer notification profile-after-change
1545821436575	Marionette	DEBUG	Received observer notification command-line-startup
1545821436575	Marionette	DEBUG	Received observer notification nsPref:changed
1545821436575	Marionette	DEBUG	Init aborted (running=false, enabled=true, finalUIStartup=false)
1545821436797	Marionette	DEBUG	Received observer notification toplevel-window-ready
Can't find symbol 'GetGraphicsResetStatus'.
1545821437813	Marionette	DEBUG	Received observer notification sessionstore-windows-restored
1545821437813	Marionette	DEBUG	Waiting for delayed startup...
1545821438036	Marionette	DEBUG	Setting recommended pref app.update.disabledForTesting to true
1545821438036	Marionette	DEBUG	Setting recommended pref security.turn_off_all_security_so_that_viruses_can_take_over_this_computer to true
1545821438037	Marionette	DEBUG	Setting recommended pref apz.content_response_timeout to 60000
1545821438037	Marionette	DEBUG	Setting recommended pref browser.contentblocking.introCount to 99
1545821438037	Marionette	DEBUG	Setting recommended pref browser.download.panel.shown to true
1545821438037	Marionette	DEBUG	Setting recommended pref browser.newtabpage.enabled to false
1545821438038	Marionette	DEBUG	Setting recommended pref browser.pagethumbnails.capturing_disabled to true
1545821438038	Marionette	DEBUG	Setting recommended pref browser.search.update to false
1545821438038	Marionette	DEBUG	Setting recommended pref browser.tabs.disableBackgroundZombification to false
1545821438039	Marionette	DEBUG	Setting recommended pref browser.tabs.warnOnCloseOtherTabs to false
1545821438039	Marionette	DEBUG	Setting recommended pref browser.tabs.warnOnOpen to false
1545821438039	Marionette	DEBUG	Setting recommended pref browser.usedOnWindows10.introURL to 
1545821438039	Marionette	DEBUG	Setting recommended pref browser.urlbar.suggest.searches to false
1545821438040	Marionette	DEBUG	Setting recommended pref datareporting.policy.dataSubmissionPolicyAccepted to false
1545821438040	Marionette	DEBUG	Setting recommended pref dom.disable_beforeunload to true
1545821438040	Marionette	DEBUG	Setting recommended pref dom.disable_open_during_load to false
1545821438040	Marionette	DEBUG	Setting recommended pref dom.file.createInChild to true
1545821438040	Marionette	DEBUG	Setting recommended pref dom.max_chrome_script_run_time to 0
1545821438040	Marionette	DEBUG	Setting recommended pref dom.max_script_run_time to 0
1545821438040	Marionette	DEBUG	Setting recommended pref extensions.getAddons.cache.enabled to false
1545821438041	Marionette	DEBUG	Setting recommended pref extensions.webservice.discoverURL to http://%(server)s/dummy/discoveryURL
1545821438041	Marionette	DEBUG	Setting recommended pref network.http.prompt-temp-redirect to false
1545821438041	Marionette	DEBUG	Setting recommended pref network.http.speculative-parallel-limit to 0
1545821438041	Marionette	DEBUG	Setting recommended pref security.fileuri.strict_origin_policy to false
1545821438041	Marionette	DEBUG	Setting recommended pref security.notification_enable_delay to 0
1545821438041	Marionette	DEBUG	Setting recommended pref signon.autofillForms to false
1545821438041	Marionette	DEBUG	Setting recommended pref signon.rememberSignons to false
1545821438041	Marionette	DEBUG	Setting recommended pref toolkit.cosmeticAnimations.enabled to false
1545821438092	Marionette	INFO	Listening on port 55241
1545821438092	Marionette	DEBUG	Remote service is active
1545821438137	Marionette	DEBUG	Accepted connection 0 from 127.0.0.1:55270
1545821438141	Marionette	TRACE	0 -> [0,1,"WebDriver:NewSession",{"browserName":"firefox","capabilities":{"desiredCapabilities":{"browserName":"firefox"}}}]
1545821438206	Marionette	DEBUG	[4294967297] Frame script loaded
1545821438208	Marionette	DEBUG	[4294967297] Frame script registered
1545821438216	Marionette	TRACE	0 <- [1,1,null,{"sessionId":"6f533b1f-2320-eb4b-b1a9-79ddf52406e6","capabilities":{"browserName":"firefox","browserVersion":"64.0" ... ecCompliantPointerOrigin":false,"moz:webdriverClick":true,"capabilities":{"desiredCapabilities":{"browserName":"firefox"}}}}]
1545821438234	Marionette	TRACE	0 -> [0,2,"WebDriver:Navigate",{"url":"https://webdriver.io"}]
1545821438245	Marionette	DEBUG	[4294967297] Received DOM event beforeunload for about:blank
1545821438767	Marionette	DEBUG	[4294967297] Received DOM event pagehide for about:blank
1545821441278	Marionette	DEBUG	[4294967297] Received DOM event DOMContentLoaded for https://webdriver.io/
1545821449271	Marionette	DEBUG	[4294967297] Received observer notification outer-window-destroyed
1545821449271	Marionette	DEBUG	[4294967297] Received observer notification outer-window-destroyed
1545821449274	Marionette	DEBUG	[4294967297] Received observer notification outer-window-destroyed
1545821449751	Marionette	DEBUG	[4294967297] Received DOM event pageshow for https://webdriver.io/
1545821449752	Marionette	TRACE	0 <- [1,2,null,{"value":null}]
1545821449759	Marionette	TRACE	0 -> [0,3,"WebDriver:GetTitle",{}]
1545821449760	Marionette	TRACE	0 <- [1,3,null,{"value":"WebdriverIO · Next-gen WebDriver test framework for Node.js"}]
console.error: BroadcastService: 
  receivedBroadcastMessage: handler for
  remote-settings/monitor_changes
  threw error:
  Message: Error: Polling for changes failed: NetworkError when attempting to fetch resource..
  Stack:
    remoteSettingsFunction/remoteSettings.pollChanges@resource://services-settings/remote-settings.js:750:13
```

### PR DESCRIPTION
**Pull request by goofballLogic**
## Proposed changes

This is a similar change to the one needed for wdio-cli (https://github.com/webdriverio/webdriverio/commit/5f506af894e2d06268c576b7a8cd32220fe2c754) which enables us to run scripts which use wdio in standalone mode (and allows the Getting Started page to work).

Fixes #3182 

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

Note that I haven't added a unit test as I think this would require an integration test to catch it.

### CODE REVIEW

**Comment by goofballLogic** [comment]
Ok, but the line of code which requires this package is reported as webdriverio/build/index.js:8:1. I don't understand the build mechanism, but I was guessing that the dependency is `webdriverio`&rarr;`source-map-support` rather than `webdriver`&rarr;`source-map-support` ?

File: packages/webdriverio/package.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "lodash.zip": "^4.2.0",
     "rgb2hex": "^0.1.0",
     "serialize-error": "^3.0.0",
+    "source-map-support": "^0.5.6",
```
Comment (cuki) [inline packages/webdriverio/package.json]: Why version __^0.5.6__, when `devDependencies` has version __^0.5.9__?
https://github.com/webdriverio/webdriverio/blob/c31a3a6583120ede3892b5248488eb34641c2395/package.json#L52

Or are you simply moving the dependency from wdio-cli to webdriver(io)?
https://github.com/webdriverio/webdriverio/blob/c31a3a6583120ede3892b5248488eb34641c2395/packages/wdio-cli/package.json#L52

Comment (goofballLogic) [reply packages/webdriverio/package.json]: I simply tried to synchronise the dependency between the CLI and the standalone packages. Someone more qualified than I would have to advise about which version is preferable.

**Comment by cuki** [comment]
> Ok, but the line of code which requires this package is reported as webdriverio/build/index.js:8:1. I don't understand the build mechanism, but I was guessing that the dependency is `webdriverio`→`source-map-support` rather than `webdriver`→`source-map-support` ?

@goofballLogic This is because [__babel-plugin-source-map-support__](https://www.npmjs.com/package/babel-plugin-source-map-support) prepends the following statement to source files:
```
import 'source-map-support/register';
```

It seems that whichever module is using the babel plugin in `devDependencies` should have the associated __source-map-support__ in `dependencies` as instructed on the [package page](https://www.npmjs.com/package/babel-plugin-source-map-support):
```
$ npm install source-map-support --save
$ npm install babel-plugin-source-map-support --save-dev
```

**Comment by goofballLogic** [comment]
> It seems that whichever module is using the babel plugin in devDependencies should have the associated source-map-support in dependencies as instructed on the package page:

@cuki by that logic, this dependency ought to be in the top level package.json where the babel plugin is registered: https://github.com/webdriverio/webdriverio/blob/master/package.json#L38

I see that it's already there (but in the devDependencies section). I don't understand how the build system works. Do you think it should just be moved from devDependencies to the dependencies section, rather than what I've suggested in this PR? If so, should I also remove the one from the CLI package's package.json ?

**Comment by cuki** [comment]
> > It seems that whichever module is using the babel plugin in devDependencies should have the associated source-map-support in dependencies as instructed on the package page:
> 
> @cuki by that logic, this dependency ought to be in the top level package.json where the babel plugin is registered: https://github.com/webdriverio/webdriverio/blob/master/package.json#L38
> 
> I see that it's already there (but in the devDependencies section). I don't understand how the build system works. Do you think it should just be moved from devDependencies to the dependencies section, rather than what I've suggested in this PR? If so, should I also remove the one from the CLI package's package.json ?

@goofballLogic Yes, technically the problem should be solved by moving [__"source-map-support": "^0.5.9"__](https://github.com/webdriverio/webdriverio/blob/c31a3a6583120ede3892b5248488eb34641c2395/package.json#L52) from `devDependencies` to `dependencies`.

However I'm not sure about wdio packages which do not have __webdriverio__ as dependency, because they use the same babel compiler, but also do not have __source-map-support__ defined as dependency.

@christian-bromann Is __source-map-support__ even supposed to be in published packages?
Looking at the babel config, this is supposed to be only applicable for development:
https://github.com/webdriverio/webdriverio/blob/816bde0ab08dae111596dc202d086dba5e6020c2/babel.config.js#L15-L20

Problem is though, when no environment variable (`BABEL_ENV` or `NODE_ENV`) specified it will fallback to "development", as per documentation (https://babeljs.io/docs/en/options#envname).
Maybe when running `npm run release` it should specify a different value as "development"?

**Comment by christian-bromann** [comment]
I would like to close this in favor of https://github.com/webdriverio/webdriverio/pull/3242

**Pull request closed without merging**